src/C4Network2Dialogs.h

```diff
@@ -119,7 +119,9 @@ class C4Network2ClientListBox : public C4GUI::ListBox
 		C4GUI::Icon *pStatusIcon;  // client network status
 		C4GUI::Label *pName;       // client name
 		C4GUI::Label *pPing;       // client control ping
-		C4GUI::IconButton *pActivateBtn, *pKickBtn; // buttons for host
+		C4GUI::IconButton *pMuteBtn;     // mute client (/sound command)
```

Comment:
Hungarian notation is forbidden by the style guide, however, I'm not sure whether it's better to keep in line here or not.